### PR TITLE
Integrating fetch standup api

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue

- https://github.com/code-mancers/g-bot/issues/19

# functionalities Added or Changed

- integrated the api endpoint to frontend that fetches standup data from database
- fetched data will be displayed on the home page as a List of standups

# Checklist

- [x] Uploaded screenshots

# Screenshots
Screenshot 2022-10-18 at 11 47 50 AM

### CODE REVIEW

File: client/src/components/Standups/StandupList.tsx [hunk 1]
```diff
@@ -15,8 +15,11 @@ const StandupList = (props: ListProps) => (
     <Datagrid>
       <TextField source="id" />
       <TextField source="title" />
-      <TextField source="channel" />
-
+      <ArrayField source="channel">
+        <SingleFieldList>
+          <ChipField source="slackChannelId" />
```
Comment (SujayPrabhu96) [inline client/src/components/Standups/StandupList.tsx:20]: We need to display channel name
```suggestion
          <ChipField source="name" />
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/components/Standups/StandupList.tsx (hunk 1 above)
Comment (ayushdevelop) [reply client/src/components/Standups/StandupList.tsx:20]: was thinking of doing it in a different branch with changes for SelectInput as well, needed to read values of dummy data in this one

Comment (SujayPrabhu96) [reply client/src/components/Standups/StandupList.tsx:20]: Okay

**Review by SujayPrabhu96 — APPROVED** [review]

File: client/src/components/Standups/StandupCreate.tsx [hunk 2]
```diff
@@ -100,13 +99,13 @@ const StandupCreate = () => {
...
           source="Days"
           choices={[
-            { id: "1", name: "Monday" },
-            { id: "2", name: "Tuesday" },
-            { id: "3", name: "Wednesday" },
-            { id: "4", name: "Thursday" },
-            { id: "5", name: "Friday" },
-            { id: "6", name: "Saturday" },
-            { id: "7", name: "Sunday" },
+            { id: 1, name: "Monday" },
```
Comment (SujayPrabhu96) [inline client/src/components/Standups/StandupCreate.tsx:102]: While you are updating the channel id to name, move this array to constants/*.ts file

**Review by SujayPrabhu96 — APPROVED** [review]
One minor comment. Can be done in next PR. LGTM

File: client/src/components/Standups/StandupCreate.tsx (hunk 2 above)
Comment (ayushdevelop) [reply client/src/components/Standups/StandupCreate.tsx:102]: okay

**Pull request merged**